mbtools/html_content_extractor.py

```diff
@@ -0,0 +1,48 @@
+from mbtools import utils
+from pathlib import Path
+import argparse
+
+
+class ContentExtractor:
+    """
+    Given a string with path to an extracted moodle backup directory return
+    model objects for course html elements
+    """
+    def __init__(self, mbz_path):
+        self.mbz_path = Path(mbz_path)
+        self.output_html_files = {}
+
+    def replace_content_tags(self, output_file_path):
+        activities = utils.parse_backup_activities(self.mbz_path)
+
+        for act in activities:
+            for html_elem in act.html_elements():
+
+                self.output_html_files.update(html_elem.replace_content_tag())
+                with open(act.activity_filename, "wb") as f:
+                    act.etree.write(f, encoding="utf-8",pretty_print=True,
+                                    doctype='<?xml version="1.0" encoding="UTF-8"?>')
```

Comment:
Just out of curiosity, why are we passing `doctype` here versus using `xml_declaration=True`?

Reply:
xml_declaration=true will use single quotes for "1.0" and "UTF-8". I was trying to mimic the test string. I am still looking into why it's using single quotes. passing doctype was my temporary fix.

Reply:
Ah, I see. This may be more of a cosmetic difference since both should be fine in terms of validity. If things work end-to-end as far as importing into Moodle in practice, I would suggest we stick with whatever `lxml` is doing if we can't influence it and adjust unit tests as needed.

Reply:
Will do. Thanks